### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by MarieMinasyan**

### CODE REVIEW

**Comment by rybus** [comment]
GTM

File: contributing/feature_request.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Submit a Feature Request
 ========================
 
-* you can suggest any feature here http://www.akeneo.com/forum/feature-request/
-* after discussion, this feature could be included in our roadmap
+You can suggest any feature here http://www.akeneo.com/forum/feature-request/.
+After discussion, this feature could be included in our roadmap
```
Comment (phaseinducer) [inline contributing/feature_request.rst]: Dot at the end?

File: reference/best_practices/reusable_bundle.rst [hunk 2]
```diff
@@ -117,22 +117,22 @@ Avoid to override Akeneo PIM classes/services
...
+If both of them override this service, the last bundle listed in the `app/AppKernel.php` will determine the one that will be used.
 
 Another consequence is that it will have an impact on the whole application. That being said, it might be what you want to do.
 
 
 Models
 """"""
 You should avoid to extend models (entities and documents) in order to add properties.
-You would face the same problem than when you override classes/services.
+You would face the same problem as the ones you encounter when you override classes/services.
```
Comment (phaseinducer) [inline reference/best_practices/reusable_bundle.rst]: "the one" or "problems" (plural)

File: reference/technical_information/server_side_setup_for_hosting.rst [hunk 3]
```diff
@@ -15,7 +15,7 @@ Here is the minimum server configuration to run Akeneo PIM application:
 
 .. warning::
 
-  Should you use VCPu, keep in mind that VCPus do not provide same level of performance as for as real CPUs. Amazon servers, for instance, offer only 10 to 25% of performances of a real CPU
+  Should you use VCPu, keep in mind that VCPus do not provide same level of performance as real CPUs. Amazon servers, for instance, offer only 10 to 25% of performances of a real CPU
```
Comment (phaseinducer) [inline reference/technical_information/server_side_setup_for_hosting.rst]: Dot at the end?

**Comment by phaseinducer** [comment]
GTM after fix =)

**Pull request merged**